app/javascript/turbo/fetch_requests.ts

```diff
@@ -0,0 +1,36 @@
+import type { TurboBeforeFetchRequestEvent, TurboSubmitStartEvent } from '@hotwired/turbo'
+
+export function encodeMethodIntoRequestBody(event: Event) {
+  let ev = event as TurboBeforeFetchRequestEvent
+  if (ev.target instanceof HTMLFormElement) {
+    const { target: form, detail: { fetchOptions } } = ev
+
+    form.addEventListener("turbo:submit-start", ev => {
```

Comment:
To add what @seanpdoyle mentioned in the previous comment: I've opened https://github.com/hotwired/turbo/pull/800 to help with this.

With that you would be able to do something like:
```suggestion
export function encodeMethodIntoRequestBody(event: TurboBeforeFetchRequestEvent) {
  if (event.target instanceof HTMLFormElement) {
    const { target: form, detail: { fetchOptions } } = event

    form.addEventListener("turbo:submit-start", (ev: TurboSubmitStartEvent) => {
```

Reply:
It looks good to me. I look forward to seeing the pullreq merged 😀